src/components/navbar/Nav.jsx

```diff
@@ -0,0 +1,118 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import logo from '../logo.jpg';
+import './Nav.css';
+
+const Nav = () => {
+  const [isMobile, setIsMobile] = useState(false);
+
+
+  return (
+    <div className="navigation-panel">
+      <header>
+        <div
+          className={
+            isMobile
+              ? 'nav-icons-holder '
+              : 'nav-icons-holder nav-mobile-border'
+          }
+        >
+          <img src={logo} alt="Logo" className="logo" />
+          <div className="nav-btn-holder">
+            <Link to="/login">
+              <button className="button-sign-log" type="submit">
+                Log in
+              </button>
+            </Link>
+            <Link to="/sign-up">
+              <button className="button-sign-log" type="submit">
+                Sign up
+              </button>
+            </Link>
+            <button
+              className="button-icon"
+              type="submit"
+              onClick={() => setIsMobile(!isMobile)}
+            >
+              {isMobile ? (
+                <i className="fas fa-times fa-2x" />
+              ) : (
+                <i className="fas fa-bars fa-2x" />
+              )}
+            </button>
+          </div>
+        </div>
+      </header>
+      <nav>
+        <ul className={isMobile ? 'show-mobile-menu' : 'hide-mobile-menu'}>
+          <li className="nav-list">
+            <Link className="nav-link link-style" to="/resorts">
+              Resorts
+            </Link>
+          </li>
```

Comment:
- [ ] I think, we need to add a `Reserve` nav link as well.
![image](https://user-images.githubusercontent.com/44057863/173212849-fedeb5f6-afff-4c60-95b1-1403167c43ae.png)

- [ ] Can we give a different background color to the active link in the navigation panel? Like in the design below:
![image](https://user-images.githubusercontent.com/44057863/173213029-e71dc4e7-ae79-4cea-b178-7008a269c0c1.png)